Documentation/project-docs/dogfooding.md

```diff
@@ -0,0 +1,108 @@
+# How to patch dotnet CLI with latest CoreFX
+
+This document provides the steps necessary to consume a nightly build of CoreFX
+and CoreCLR.
+
+Please note that these steps aren't necessary for official builds -- these steps
+are specific to consuming nightlies and thus unsupported builds. Also note that
+these steps are likely to change as we're simplifying this experience. Make
+sure to consult this document often.
+
+## Install prerequisites
+
+1. Install CLI preview 4 SDK
+    - [Win 64-bit Latest](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe)
+    - [macOS 64-bit Latest](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-osx-x64.latest.dmg)
```

Comment:
Wrong link. Should be:

https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-osx-x64.latest.pkg